docs/testimonials.md

```diff
@@ -73,6 +73,32 @@ Jujutsu exists)
 
 &mdash; gul banana, computer programmer
 
+> haha, I have corrupted my cofounder and now 100% of brontosource dev is on JJ.
+> (yes, 100% is two of us)
+
+&mdash; kfm, (what to write here)
```

Comment:
```suggestion
> I have corrupted my cofounder and now 100% of brontosource dev is on JJ.

&mdash; Matt Kulukundis, cofounder BrontoSource
```

Reply:
@algmyr suggested to keep the "muwhahaha" laughter  in Discord and not all of these need to be completely serious.